### DISCUSSION TITLE
is this a common issue when resizing windows

### DISCUSSION THREAD

**Discussion by bassamsdata** [LazyVim/LazyVim#237] [Q&A] [answered]
Hi guys, I have this funny issue.

when I press <C-Down> (when I don't have horizontal splits) the lualine duplicate and goes up and then I can't return it back, so I have to close neovim and start again.

I'm posting here to see if I'm the only one who having this issue or this is common. Then I'd submit an issue on lualine repo.

useful info: OS: MacOS. terminal: Alacritty and kitty. lualine: didn't change any settings (even if I changed the settings I got the same issue)

video:

https://user-images.githubusercontent.com/105807570/218238271-da5796d7-397c-4d46-a20f-9e77a4d3279f.mov

Thanks.

**Comment by folke** [comment, marked as answer]
Other people had a similar issue, but the issue seems to be fixed on Nightly, so maybe try that or wait till it gets released.

**Reply by bassamsdata** [reply]
Thank you for your answer and sorry for the delay, I didn't have internet in the weekend. but do you mean nightly version of lualine tho?

**Reply by dpetka2001** [reply]
He meant nightly version of neovim, I'm fairly sure.

**Reply by folke** [reply]
yes, nightly Neovim.

**Reply by bassamsdata** [reply]
Thank you guys
